### PR TITLE
Added Loen-Handler

### PR DESCRIPTION
**Pull request by s7904893**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
+# -*- encoding: iso-8859-15 -*-
```
Comment (fried-ice) [inline src/main.py]: Hell nah.
UTF-8 all the way.

File: src/main.py [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 #!/bin/python
 
 from telegram.ext import Updater, CommandHandler, MessageHandler, Filters
 import requests, json
 import os
 import logging
 from datetime import date, timedelta
-
```
Comment (fried-ice) [inline src/main.py]: Please do not change line spacing.

File: src/main.py [hunk 3]
```diff
@@ -41,20 +42,32 @@ def mensa(update, context):
 
 
 def andre(update, context):
-    context.bot.send_message(chat_id=update.message.chat_id, text="Höhöhö Reichenbach!")
+    context.bot.send_message(chat_id=update.message.chat_id, text="H�h�h� Reichenbach!")
```
Comment (fried-ice) [inline src/main.py]: This is probably related to the encoding change, not needed though.

File: src/main.py [hunk 4]
```diff
@@ -41,20 +42,32 @@ def mensa(update, context):
...
-        context.bot.send_message(chat_id=update.message.chat_id, text="I failed miserably. Disgrace!")
-        return
-    context.bot.send_message(chat_id=update.message.chat_id, text=resp.text)
+        return "I failed miserably. Disgrace!"
+    return resp.text
 
 
 def maxime(update, context):
     context.bot.send_sticker(chat_id=update.message.chat_id, sticker="CAADBQADfAMAAukKyAPfAAFRgAuYdNoWBA")
+
```
Comment (fried-ice) [inline src/main.py]: This extra line is correct though, we can leave it.

**Review by fried-ice — CHANGES_REQUESTED** [review]

File: src/main.py (hunk 1 above)
Comment (s7904893) [reply src/main.py]: okok

File: src/main.py (hunk 4 above)
Comment (s7904893) [reply src/main.py]: Why is it still marked as a change, i removed the extra line?

File: src/main.py [hunk 5]
```diff
@@ -44,18 +46,29 @@ def andre(update, context):
...
-        return
-    context.bot.send_message(chat_id=update.message.chat_id, text=resp.text)
+        return "I failed miserably. Disgrace!"
+    return resp.text
 
 
 def maxime(update, context):
     context.bot.send_sticker(chat_id=update.message.chat_id, sticker="CAADBQADfAMAAukKyAPfAAFRgAuYdNoWBA")
-
+
```
Comment (fried-ice) [inline src/main.py]: Whitespace!

**Review by fried-ice — CHANGES_REQUESTED** [review]

**Pull request merged**